static/sample.env

```diff
@@ -0,0 +1,2 @@
+MAPBOX_KEY=[[REPLACE_ME]]
```

Comment:
can we rename all these keys to `<provider>_API_KEY` instead of `<provider>_KEY`? that way it'll match our other repos and if did want to pass these tokens to our reusable GH workflows, we wouldn't have to switch names

also, can we add the google and leaflet api keys to both sample.env files?